node/lib/package/PackageInfo.ts

```diff
@@ -181,3 +184,25 @@ export class PackageTranslatorInfo {
      */
     public readonly onboardingProperties: { [propertyName: string]: string };
 }
+
+export class PackageOnboardingInfo
+{
+    public readonly onboardingId: string;
```

Comment:
Yes, referring to onboardingId.

---
In reply to: [75343685](https://github.com/openT2T/opent2t/pull/22#discussion_r75343685) [](ancestors = 75343685,75262878)